MPF.Modules/BaseParameters.cs

```diff
@@ -1396,6 +1396,43 @@ protected static string GetPlayStation5Version(char? driveLetter)
             }
         }
 
+        /// <summary>
+        /// Get the internal serial from a PlayStation 4 disc, if possible
+        /// </summary>
+        /// <param name="driveLetter">Drive letter to use to check</param>
+        /// <returns>Internal disc serial if possible, null on error</returns>
+        protected static string GetPlayStation4Serial(char? driveLetter)
```

Comment:
Minor nit: please move this alphabetically. In this case after `GetPlayStation4Version` and before `GetPlayStation5Version`.